src/__tests__/__snapshots__/main-test.js.snap

```diff
@@ -267,6 +267,35 @@ Object {
   "displayName": "MyComponent",
   "methods": Array [],
   "props": Object {
+    "OtherComponentProps": Object {
+      "description": "Spread props from another file",
+      "flowType": Object {
+        "name": "OtherComponentProps",
+      },
+      "inherited": true,
```

Comment:
These have a flag for whether it is inherited. This was cleaner than adding a new top level key to the output documentation.